src/main/java/io/mosip/mimoto/controller/IssuersController.java

```diff
@@ -59,18 +59,12 @@ public ResponseEntity<Object> getAllIssuers(@RequestParam(required = false) Stri
 
     @GetMapping("/{issuer-id}/wellknown")
     public ResponseEntity<Object> getIssuerWellknown(@PathVariable("issuer-id") String issuerId) {
-        ResponseWrapper<Object> responseWrapper = new ResponseWrapper<>();
-        responseWrapper.setId(ID);
-        responseWrapper.setVersion("v1");
-        responseWrapper.setResponsetime(DateUtils.getRequestTimeString());
         try {
             CredentialIssuerWellKnownResponse credentialIssuerWellKnownResponse = issuersService.getIssuerWellknown(issuerId);
-            responseWrapper.setResponse(credentialIssuerWellKnownResponse);
-            return ResponseEntity.status(HttpStatus.OK).body(responseWrapper);
+            return ResponseEntity.status(HttpStatus.OK).body(credentialIssuerWellKnownResponse);
         } catch (Exception exception) {
             logger.error("Exception occurred while fetching issuers wellknown ", exception);
-            responseWrapper = handleExceptionWithErrorCode(exception);
-            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(responseWrapper);
+            return ResponseEntity.status(HttpStatus.BAD_REQUEST).body(null);
```

Comment:
Inji mobile and Inji Web is expects a complete wellknown to be received as a part of this, instead of null can we return the fallback wellknown, or a proper error response while UI can handle gracefully.

Reply:
fallback wellknown is part of config server, so it will be a api rest. If it fails to fetch, what can be the fallback json?
UI can handle the response as wellknownResponse exists or not?

Reply:
or we can return HttpStatus.NOT_FOUND instead of BAD_REQUEST. while will let us know the resource is not found.